diff -rupN --no-dereference scotch_6.0.10/src/esmumps/Makefile scotch_6.0.10-new/src/esmumps/Makefile
--- scotch_6.0.10/src/esmumps/Makefile	2020-08-30 20:55:50.000000000 +0200
+++ scotch_6.0.10-new/src/esmumps/Makefile	2020-09-02 10:24:41.145309123 +0200
@@ -177,7 +177,7 @@ esmumps_strats$(OBJ)		:	esmumps_strats.c
 					esmumps.h
 
 libesmumps$(LIB)		:	$(LIBESMUMPSDEPS)
-					$(AR) $(ARFLAGS) $(@) $(?)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS)
 					-$(RANLIB) $(@)
 
 main_esmumps$(EXE)		:	main_esmumps.c				\
diff -rupN --no-dereference scotch_6.0.10/src/libscotch/Makefile scotch_6.0.10-new/src/libscotch/Makefile
--- scotch_6.0.10/src/libscotch/Makefile	2020-01-12 20:49:29.000000000 +0100
+++ scotch_6.0.10-new/src/libscotch/Makefile	2020-09-02 10:24:41.146309161 +0200
@@ -3114,26 +3114,26 @@ ptscotchf.h			:	ptdummysizes$(EXE)			\
 					library_pt_f.h
 					./ptdummysizes$(EXE) "-s$(SCOTCH_NAME_SUFFIX)" library_pt_f.h ptscotchf.h
 
-libscotch$(LIB)			:	$(LIBSCOTCHDEPS)
-					$(AR) $(ARFLAGS) $(@) $(?)
+libscotch$(LIB)			:	$(LIBSCOTCHDEPS) libscotcherr$(LIB)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS) -L . -lscotcherr
 					-$(RANLIB) $(@)
 
 libscotcherr$(LIB)		:	library_error$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(?)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS)
 					-$(RANLIB) $(@)
 
 libscotcherrexit$(LIB)		:	library_error_exit$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(?)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS)
 					-$(RANLIB) $(@)
 
-libptscotch$(LIB)		:	$(LIBPTSCOTCHDEPS)
-					$(AR) $(ARFLAGS) $(@) $(?)
+libptscotch$(LIB)		:	$(LIBPTSCOTCHDEPS) libscotch$(LIB)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS) -L . -lscotch
 					-$(RANLIB) $(@)
 
 libptscotcherr$(LIB)		:	library_error_pt$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(?)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS)
 					-$(RANLIB) $(@)
 
 libptscotcherrexit$(LIB)	:	library_error_exit_pt$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(?)
+					$(AR) $(ARFLAGS) $(@) $(?) $(LDFLAGS)
 					-$(RANLIB) $(@)
diff -rupN --no-dereference scotch_6.0.10/src/libscotchmetis/Makefile scotch_6.0.10-new/src/libscotchmetis/Makefile
--- scotch_6.0.10/src/libscotchmetis/Makefile	2019-05-19 01:42:19.000000000 +0200
+++ scotch_6.0.10-new/src/libscotchmetis/Makefile	2020-09-02 10:24:41.147309199 +0200
@@ -142,12 +142,12 @@ libptscotchparmetis$(LIB)	:	parmetis_dgr
 					parmetis_dgraph_order_f$(OBJ)		\
 					parmetis_dgraph_part$(OBJ)		\
 					parmetis_dgraph_part_f$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(^)
+					$(AR) $(ARFLAGS) $(@) $(^) $(LDFLAGS) -L ../libscotch -lptscotch
 					-$(RANLIB) $(@)
 
 libscotchmetis$(LIB)		:	metis_graph_order$(OBJ)			\
 					metis_graph_order_f$(OBJ)		\
 					metis_graph_part$(OBJ)			\
 					metis_graph_part_f$(OBJ)
-					$(AR) $(ARFLAGS) $(@) $(^)
+					$(AR) $(ARFLAGS) $(@) $(^) $(LDFLAGS) -L ../libscotch -lscotch
 					-$(RANLIB) $(@)
